pkg/router/template/router.go

```diff
@@ -785,6 +785,12 @@ func cmpStrSlices(first []string, second []string) bool {
 // it will log a warning.  The route will still be written but users may receive browser errors
 // for a host/cert mismatch
 func (r *templateRouter) shouldWriteCerts(cfg *ServiceAliasConfig) bool {
+
+	// The cert is already written
+	if cfg.Status == ServiceAliasConfigStatusSaved {
```

Comment:
Do we ever set the status to this when there are no certs?  We return false if they don't have certs, and that means we never call WriteCertificatesForConfig.

So, do we need to set cfg.Status in this function?  If so, should we re-use the same state or make a new one "Skipped" or something?